### PR TITLE
Add Pycalverter

### PR DESCRIPTION
**Pull request by Matt-BF**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by Matt-BF** [comment]
@conda-forge/help-python I believe this is all set!

File: recipes/pycalverter/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+test:
+  imports:
+    - calverter
+
+about:
+  home: https://bitbucket.org/bayazee/pycalverter/overview
+  license: GNU General Public (GPL)
+  license_family: LGPL
+  license_file:
```
Comment (marcelotrevisani) [inline recipes/pycalverter/meta.yaml]: Please change it to ```license_file: LICENSE.txt```
could you also raise an issue in this project to package the license?

File: recipes/pycalverter/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - calverter
+
+about:
+  home: https://bitbucket.org/bayazee/pycalverter/overview
+  license: GNU General Public (GPL)
```
Comment (marcelotrevisani) [inline recipes/pycalverter/meta.yaml]: Please, change it to ```license: GPL-2.0```

Comment (Matt-BF) [reply recipes/pycalverter/meta.yaml]: Changed it to `GPL2` as building with `GPL-2.0` failed

File: recipes/pycalverter/meta.yaml (hunk 1 above)
Comment (Matt-BF) [reply recipes/pycalverter/meta.yaml]: Done! Issue is #8557

File: recipes/pycalverter/meta.yaml (hunk 2 above)
Comment (marcelotrevisani) [reply recipes/pycalverter/meta.yaml]: Strange, I checked and the short version should be ``GPL-2.0``

Comment (marcelotrevisani) [reply recipes/pycalverter/meta.yaml]: But that is ok ``GPL2`` then

**Pull request merged**